UnitTests/GTMSessionFetcherFetchingTest.m

```diff
@@ -1649,7 +1639,17 @@ - (void)internalCancelFetchWithCallback:(unsigned int)sleepTime
 #pragma clang diagnostic pop
   if (!_isServerRunning) return;
 
-  CREATE_START_STOP_NOTIFICATION_EXPECTATIONS(1, 1);
+  // If the authorizer is async, then the fetch won't fully begin, and there won't ever be
+  // a start notification (and thus stop notification).
+  int expectedNotifications = ((TestAuthorizer*)authorizer).isAsync ? 0 : 1;
```

Comment:
expectedNotificationsCount? Just to make it clearer what this is recording.. as opposed to an array of expectations

Reply:
Done